guides/common/modules/con_smartproxy-features.adoc

```diff
@@ -0,0 +1,78 @@
+[id="{SmartProxy}-Features_{context}"]
+= {SmartProxy} features
+
+ifdef::satellite,orcharhino[]
+{SmartProxyServers} provide local host management services and can mirror content from {ProjectServer}.
+endif::[]
+ifndef::satellite,orcharhino[]
+{SmartProxyServers} provide local host management services.
+With the Katello plug-in, they can also mirror content from {ProjectServer}.
+endif::[]
+
+ifdef::satellite,orcharhino[]
+To mirror content from {ProjectServer}, {SmartProxy} provides the following functionalities:
+endif::[]
+ifndef::satellite,orcharhino[]
+If you have the Katello plug-in installed, you can use {SmartProxy} to mirror content from {ProjectServer}:
+endif::[]
+
+Repository synchronization::
+{SmartProxyServers} pull content for selected lifecycle environments on the {ProjectServer} and make this content available to the hosts they manage.
```

Comment:
```suggestion
{SmartProxyServers} pull content for selected lifecycle environments from {ProjectServer} and make this content available to the hosts they manage.
```

IMHO we try to avoid saying "the {Project...}".